mscore/scoreview.cpp

```diff
@@ -1112,11 +1117,41 @@ void ScoreView::paint(const QRect& r, QPainter& p)
                   }
             }
 
+
+      // AvsOmr -----
+#ifdef AVSOMR
+      Avs::AvsOmrDrawer omrDrawer;
+      std::shared_ptr<Avs::AvsOmrDrawer::Context> omrDrawCtx = omrDrawer.makeContext(&p, _score);
+      auto pageMeasures = [](Page* page) -> QList<const Measure*> {
+            QList<const Measure*> ml;
+            for (const System* s : page->systems()) {
+                  const std::vector<MeasureBase*>& measures = s->measures();
+                  for (const MeasureBase* mb : measures) {
+                        if (mb->isMeasure())
+                              ml << toMeasure(mb);
+                        }
+                  }
+            return ml;
+      };
+#endif
+      // ------------
+
       QRegion r1(r);
       if ((_score->layoutMode() == LayoutMode::LINE) || (_score->layoutMode() == LayoutMode::SYSTEM)) {
             if (_score->pages().size() > 0) {
+
                   Page* page = _score->pages().front();
                   QList<Element*> ell = page->items(fr);
+
+                  // AvsOmr -----
+#ifdef AVSOMR
+                  if (omrDrawCtx) {
+                      QList<const Measure*> ml = pageMeasures(page);
+                      omrDrawer.draw(omrDrawCtx, ml);
+                  }
```

Comment:
Still 4-space indent and non-banner-style here ;-)

Reply:
fixed